### PR TITLE
[Model][Experimental ]Support DeepSeek-V3 Pre-train

### PR DESCRIPTION
**Pull request by lxd-cumt**
# Description
🎉 We are excited to announce the release of DeepSeek-V3 Pretraining! This pull request (PR) includes a test configuration, training scripts, and the base module.

The training pipeline has been validated, and its key features are as follows:
- Support for checkpoint conversion (Hugging Face to Megatron) for warm-start training.
- Support for MLA, shared experts, sigmoid routers, and loss-free load balancing based on the latest Megatron framework.
- Support for parallel strategies, including data parallelism, tensor parallelism, pipeline parallelism, and expert parallelism.

# Work in Progress
- [ ] MTP Module and Distributed Training (coming soon)
- [ ] FP8 Training
- [ ] DualPipe Schedule
- [ ] Overlap and Memory Optimization
- [ ] Support for More Chips

### CODE REVIEW

**Review by aoyulong — APPROVED** [review]
LGTM. Good job!
